variable/technology/technologies.yaml

```
@@ -1089,3 +1089,10 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a Wind OffShore power plant
    unit: '%'
+   
+# description of Demand flexibility potentials
+   
+Load Shifting Potential|Electricity|Residential|AC:
```

I'd suggest to spell it out to avoid confusion with "Alternate Current"
```suggestion
Load Shifting Potential|Electricity|Residential|Air Conditioning:
```